syncstorage/tests/functional/test_storage.py

```diff
@@ -1964,6 +1965,15 @@ def test_that_we_dont_resurrect_committed_batches(self):
             req = '/storage/xxx_col1?batch=true'
             resp = self.retry_post_json(self.root + req, bsos)
             batch1 = resp.json['batch']
+            try:
+                uuid.UUID(batch1)
+            except ValueError:
+                pass
+            else:
+                # On the Spanner backend: batch ids are uuid4s which
+                # should almost never collide. Plus this test will run
+                # too long, potentially expiring auth tokens
+                raise unittest2.SkipTest('N/A to Spanner backend')
```

Comment:
I'm a bit confused on this. We check to see if `batch1` is a valid UUID, ignore if it isn't, and skip the test if there's any other problem with `batch1`?

Seems like kind of a weird fail no-op. 

Maybe a bit of info around what sort of "other" error you're skipping might help.

Reply:
I interpreted this as "Skip the test if `batch` is a valid uuid, because this indicates the spanner backend, and the test provides no value for spanner"; if accurate then it may be worth refactoring the comment to indicate that at the start of the `try` block rather than inline.